.ci/localhost_ansible_tests.py

```diff
@@ -20,12 +20,15 @@
 
 
 with ci_lib.Fold('job_setup'):
-    # Don't set -U as that will upgrade Paramiko to a non-2.6 compatible version.
-    run("pip install -q virtualenv ansible==%s", ci_lib.ANSIBLE_VERSION)
```

Comment:
Where did the installation of ansible move to? I don't see a corresponding change in `localhost_ansible_install.py`.

Reply:
It was already there: https://github.com/dw/mitogen/pull/715/files#diff-ba88d108dcb714bf83179965a30e86a499f9b9e24b87d21592d40387f8394d76L14 😅 I deleted a duplicate install line

Reply:
ok

Reply:
I think this should be one of the test fixes that is split out from this PR.